hucssley-classes.md

```diff
@@ -612,6 +612,16 @@ $hu-appearance-types: (
 );
 ```
 
+### hu-backdrop-filter
```

Comment:
Can you also update the Table of Contents?

Reply:
Will do! Thanks.